content-tree.d.ts

```diff
@@ -252,23 +252,28 @@ export declare namespace ContentTree {
         children: [TableCaption, TableBody, TableFooter] | [TableCaption, TableBody] | [TableBody, TableFooter] | [TableBody];
         columnSettings: TableColumnSettings[];
     }
-    interface CustomCodeComponent extends Parent {
+    interface CccFallbackText extends Node {
+        type: 'ccc-fallback-text';
+        children: Paragraph[];
+    }
+    type CccAttributes = {
+        [key: string]: string | boolean | undefined;
+    };
+    interface CustomCodeComponent extends Node {
         type: "custom-code-component";
+        id: string;
+        embedded: boolean;
```

Comment:
Suggestion: as per my previous comments we should document every field. Not sure why it's needed in multiple places, is it automatically generated in the readme from here?

Reply:
Yes, automatically generated